src/Dialogs/Imap/ImapDialog.vala

```diff
@@ -171,10 +197,12 @@ public class OnlineAccounts.ImapDialog : Hdy.Window {
         smtp_server_grid.attach (smtp_revealer, 0, 3, 2);
         smtp_server_grid.attach (smtp_url_label, 0, 4);
         smtp_server_grid.attach (smtp_server_entry, 1, 4);
-        smtp_server_grid.attach (smtp_encryption_label, 0, 5);
-        smtp_server_grid.attach (smtp_encryption_combobox, 1, 5);
-        smtp_server_grid.attach (smtp_port_label, 0, 6);
-        smtp_server_grid.attach (smtp_port_spin, 1, 6);
+        smtp_server_grid.attach (smtp_port_label, 0, 5);
+        smtp_server_grid.attach (smtp_port_spin, 1, 5);
+        smtp_server_grid.attach (smtp_encryption_label, 0, 6);
+        smtp_server_grid.attach (smtp_encryption_combobox, 1, 6);
+        smtp_server_grid.attach (smtp_auth_label, 0, 7);
+        smtp_server_grid.attach (smtp_auth_combobox, 1, 7);
 
         var smtp_sizegroup = new Gtk.SizeGroup (Gtk.SizeGroupMode.HORIZONTAL);
```

Comment:
Don't forget to add `smtp_auth_label` to the sizegroup here to keep things aligned :)